### PR TITLE
Update VDatePicker.ts

### PR DESCRIPTION
**Pull request by doox911**
Optimized work. Extra calls .split()

### CODE REVIEW

**Review by KaelWD — DISMISSED** [review]
split is fast and these are only called on click so the performance difference is practically nothing, it is a bit cleaner though

**Review by adamkasper — APPROVED** [review]

File: packages/vuetify/src/components/VDatePicker/VDatePicker.ts [hunk 1]
```diff
@@ -352,8 +352,11 @@ export default mixins(
       }
     },
     monthClick (value: string) {
-      this.inputYear = parseInt(value.split('-')[0], 10)
-      this.inputMonth = parseInt(value.split('-')[1], 10) - 1
+      const [year, month] = value.split('-')
+
+      this.inputYear = parseInt(year, 10)
```
Comment (adamkasper) [inline packages/vuetify/src/components/VDatePicker/VDatePicker.ts:357]: what better to use `+year` instead of `parseInt()`?

Comment (doox911) [reply packages/vuetify/src/components/VDatePicker/VDatePicker.ts:357]: Becouse: https://stackoverflow.com/questions/17106681/parseint-vs-unary-plus-when-to-use-which/17106702.

We need integer.

Comment (KaelWD) [reply packages/vuetify/src/components/VDatePicker/VDatePicker.ts:357]: Other places use `value.split('-').map(Number)` but this is fine

**Comment by KaelWD** [comment]
You still have lint errors

File: packages/vuetify/src/components/VDatePicker/VDatePicker.ts (hunk 1 above)
Comment (doox911) [reply packages/vuetify/src/components/VDatePicker/VDatePicker.ts:357]: @KaelWD When to release?)

**Comment by doox911** [comment]
@KaelWD what happend?)) I just removed the spaces, and I don't see the errors of lint or test

**Pull request merged**